module/VuFindSearch/src/VuFindSearch/Backend/LibGuides/Connector.php

```diff
@@ -233,7 +262,7 @@ protected function prepareParams(array $params)
                 'list_format' => 1,
                 'output_format' => 1,
                 'load_type' => 2,
-                'enable_description' => 0,
+                'enable_description' => 1,
```

Comment:
In theory this is a breaking change, but in practice changing this value adds more data to the search widget (API) response that was previously ignored, so there is no practical impact.  The description is not actually displayed unless the config property is set, so there is no visible effect by default.

Reply:
No objection, though it seems it would be easy to set this based on the need for descriptions if you wanted to (e.g. to save a little bit of bandwidth).

Reply:
Agreed & done.